src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1488,5 +1488,9 @@ internal enum ErrorCode
         ERR_PatternWrongGenericTypeInVersion = 8314,
 
         #endregion diagnostics introduced for C# 7.1
+
+        #region diagnostics introduced for C# 7.2
+        ERR_FeatureNotAvailableInVersion7_2 = 8320,
```

Comment:
Since this is going into master, and there are no concers of conflicts, we can just use 8315?